client/src/components/ListOfProjects/ListOfProjects.css

```diff
@@ -18,7 +18,7 @@
   margin-bottom: 15px;
 }
 
-li:hover{
+li.listOrgLoggedIn:hover {
   box-shadow: 0 3px 4px 1px gray;
   cursor: pointer;
 }
```

Comment:
We should try to use the css modules pattern of:
```css
.listOrgLoggedIn {
  composes: listOrg;
}

.listOrgLoggedIn:hover {
 ...
}
```
In addition to opting for the `composes` syntax, note that the `li.` is not needed in the hover css.